tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1532,4 +1532,90 @@ public function theResponseShouldContainTheFollowingDriveInformation(TableNode $
 			throw new Error('Response is not an array or the array does not consist key "drive"');
 		}
 	}
+
+	/**
+	 * @When user :user gets details of a group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfAGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($this->getArrayOfGroupsResponded($this->listGroups($user)), $groupName);
+		$credentials = $this->getAdminOrUserCredentials($user);
+
+		$this->featureContext->setResponse(
+			GraphHelper::getSingleGroup(
+				$this->featureContext->getBaseUrl(),
+				$groupId,
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"]
+			)
+		);
+	}
+
+	/**
+	 * Get Group Id from response
+	 *
+	 * @param array|null $allGroups
+	 * @param string $groupName - name of the group
+	 *
+	 * @return string - id of the group
+	 * @throws Exception | GuzzleException
+	 */
+	public function getGroupIdByName(?array $allGroups, string $groupName): string {
```

Comment:
optional params must be after the required ones
```suggestion
	 * @param string $groupName - name of the group
	 * @param array|null $allGroups
	 *
	 * @return string - id of the group
	 * @throws Exception | GuzzleException
	 */
	public function getGroupIdByName(string $groupName, ?array $allGroups): string {
```